diskdict/batch.py

```diff
@@ -0,0 +1,44 @@
+from multiprocessing import Pool
+
+SIZE = 5000
+JOBS = 1000
+
+def _chunkify(l, n):
+    return [l[i:i + n] for i in range(0, len(l), n)]
+
+def _get(pindices):
+    words = [xd.get(i, xdefault) for i in pindices]
+    return words
+
+# NOTE: This makes 3 times fast when the
+# `k` is at the length of 100000 comparing
+# with the normal list comprehension
+def submit(ad, aindices, default=None):
```

Comment:
@RamanjaneyuluIdavalapati Did this approach increase performance? I remember there being some confusion in this area.

Reply:
@prashanthellina This approach worked.

But this uses all the cores. You have not recommended this because diskdict is a light weight module.